tests/qnn/test_cost.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+import pennylane as qml
+import pytest
+
+
+def rx_ansatz(phis, **kwargs):
+    for w, phi in enumerate(phis):
+        qml.RX(phi, wires=w)
+
+
+def layer_ansatz(weights, x=None, **kwargs):
+    qml.templates.AngleEmbedding(x, wires=[0, 1, 2])
+    qml.templates.StronglyEntanglingLayers(weights, wires=[0, 1, 2])
+
+
+@pytest.mark.parametrize("interface", qml.qnodes.decorator.ALLOWED_INTERFACES)
```

Comment:
```suggestion
@pytest.mark.parametrize("interface", qml.qnodes.decorator.ALLOWED_INTERFACES)
@pytest.mark.usefixtures("skip_if_no_torch_support", "skip_if_no_tf_support")
```
This should fix the failing tests :smile: